cli/tests/test_init_command.rs

```diff
@@ -70,3 +75,18 @@ fn test_init_local() {
     [exit status: 2]
     ");
 }
+
+// TODO: remove in jj 0.33+
+#[test]
+fn test_init_native_deprecation_warning() {
+    let test_env = TestEnvironment::default();
+    test_env.add_config(r#"ui.allow-init-native = true"#);
+    let output = test_env.run_jj_in(".", ["init", "repo"]);
+    insta::assert_snapshot!(output, @r"
+    ------- stderr -------
+    Config error: Value not found for ui.allow-init-toy
+    For help, see https://jj-vcs.github.io/jj/latest/config/.
+    [EOF]
+    [exit status: 1]
+    ");
+}
```

Comment:
I failed to only get the deprecation message here, so any thoughts on how to achieve it?